crates/mdbook-aquascope/src/preprocessor.rs

```diff
@@ -93,6 +93,11 @@ impl AquascopePreprocessor {
 
       cmd.arg(operation);
 
+      let show_flows = block.config.iter().any(|(k, _)| k == "showFlows");
+      if show_flows {
+        cmd.arg("--show-flows");
```

Comment:
Remind me again why we're gating this? Is it because you're worried it will show up a bunch otherwise?

Reply:
I don't have all the examples currently in the book memorized but I believe this feature would include new flow permission icons in some of them, and it seems unwise to include icons for a concept you aren't ready to introduce yet.